src/Application/Actions/CreatePasswordResetToken.php

```diff
@@ -8,14 +8,33 @@
 use BigGive\Identity\Repository\PasswordResetTokenRepository;
 use BigGive\Identity\Repository\PersonRepository;
 use Laminas\Diactoros\Response\JsonResponse;
+use OpenApi\Annotations as OA;
 use Psr\Http\Message\ResponseInterface as Response;
 use Psr\Http\Message\ServerRequestInterface as Request;
 use Psr\Log\LoggerInterface;
 use Slim\Exception\HttpBadRequestException;
 use Symfony\Component\Validator\Constraints\Email;
 use Symfony\Component\Validator\Validator\ValidatorInterface;
 
-class CreatePasswordResetToken extends Action
+/**
+ *  @OA\Post(
+ *     path="/v1/password-reset-token",
+ *      @OA\RequestBody(
+ *         description="",
+ *         required=true,
+ *         @OA\JsonContent(
+ *              @OA\Property(property="email_address", type="string", example="fred@example.com"),
+ *        )
+ *      ),
+ *     @OA\Response(
+ *         response=200,
```

Comment:
As above, I think where the action specifically has logic branches which 400, it's good to cover that in the doc